clientgui/DlgAdvPreferencesBase.cpp

```diff
@@ -37,6 +37,8 @@
 #define STATICBOXVERTICALSPACER 10
 #define DAYOFWEEKBORDERSIZE 10
 
+bool usingLocalPrefs;
```

Comment:
Make this a member variable instead (move to header inside `class` and rename to `m_bUsingLocalPrefs`).

Reply:
I'm afraid I've completely failed to work out how to add a boolean value to the class definition - I only need to export one lousy bit! It should be wxBool or something, but searching the online manual only finds wxBook or books about wool.